main/icicle-repl.hs

```diff
@@ -39,7 +39,7 @@ runRepl
     loop state
       = do line <- HL.getInputLine "> "
            case line of
-             Nothing      -> loop state
+             Nothing      -> return ()
```

Comment:
with this, can `CommandBlank` (where user doesn't input anything) ever happen? if not I'll remove the case.